finder-app/finder.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+filesdir=$1
+searchstr=$2
+
+if [ $# -ne 2 ]
+then
+	echo "INvalid number of arguments. Total number of arguments shoudl be 2"
```

Comment:
Can consider making this error message more descriptive to suggest the example invocation of the script.